tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -160,6 +160,10 @@ class TrtConversionParams(
       inputs during runtime, then a new TensorRT engine is built at runtime if
       allow_build_at_runtime=True, and otherwise native TF is used. This
       argument is only effective if is_dynamic_op=True.
+    allow_mixed_precision_on_unconverted_ops: whether to allow TensorFlow to
+      activate automatic mixed precision on the operations which are not
+      converted to inside a TensorRT engine. This argument is only effective if
+      the requested `precision_mode` is FP16 or INT8 and ignored for FP32.
```

Comment:
I got some feedback from internal API owners, can we change the comment to this:
whether to allow TensorFlow to use lower precisions on the operations that are not converted to inside a TensorRT engine. This argument has a default value of True, and is only effective if the requested `precision_mode` is lower than FP32.

Reply:
Should be fixed ;)